cms/views/base.py

```diff
@@ -66,12 +67,14 @@ def get_or_create_ga_client_id(self):
 
     def track_pageview(self):
         profile_id = self.settings.get('ga.profile_id')
-        if profile_id:
-            send_ga_pageview.delay(
-                profile_id,
-                self.get_or_create_ga_client_id(),
-                self.request.path,
-                self.request.remote_addr,
-                self.request.referer or '',
-                self.request.domain,
-                self.request.user_agent)
+        if not profile_id:
+            return
+
+        pageview.delay(profile_id, self.get_or_create_ga_client_id(), {
+            'path': self.request.path,
+            'uip': self.request.remote_addr,
+            'dr': self.request.referer or '',
+            'dh': self.request.domain,
+            'user_agent': self.request.user_agent,
+            'ul': self.request.accept_language,
```

Comment:
I think it's important to track both. I'll be tracking "our" locale separate as a custom variable in a separate PR.